nest/nest.py

```diff
@@ -1331,6 +1331,10 @@ def snapshot_url(self):
         else:
             return SIMULATOR_SNAPSHOT_PLACEHOLDER_URL
 
+    @property
```

Comment:
If this is not directly related to the streaming API, please submit changes into a pull request in the main repo.